src/Prince.php

```diff
@@ -1233,6 +1235,17 @@ public function setPdfId($pdfId)
         $this->pdfId = $pdfId;
     }
 
+    /**
+     * Include an AcroJS script to run when the PDF is opened.
+     *
+     * @param string $pdfScript The AcroJS script to run.
```

Comment:
Perhaps needs to be clearer that pdfScript is a filename or URL?

Reply:
Good point; done.